### PR TITLE
Heading update in installation documentation.

### LINKED ISSUE #358: Make Verify certificate order (private CA only) stand out more
**Affected Document:**

* https://docs.astronomer.io/enterprise/0.26/install-azure/
* https://docs.astronomer.io/enterprise/0.26/install-aws
* https://docs.astronomer.io/enterprise/0.26/install-gcp/

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

Verify certificate order (private CA only)

Could or should be changed to something like:

### If using a Company Created Certificate Authority

**Additional Notes:**

Would help enterprises know the order and that tls.crt should contain those items

### PR DESCRIPTION
**Pull request by kwcanuck**
This edit resolves #358.

### CODE REVIEW

File: software_versioned_docs/version-0.26/install-aws-standard.md [hunk 1]
```diff
@@ -111,21 +111,21 @@ This command will generate a report. If the `X509v3 Subject Alternative Name` se
...
 
-This command will generate a report of all certificates included. Verify that the order of these certificates is as follows:
+The command generates a report of all certificates. Verify the order of the certificates is as follows:
 
-1. Domain
-2. Intermediate (optional)
-3. Root
+- Domain
+- Intermediate (optional)
+- Root
```
Comment (jwitz) [inline software_versioned_docs/version-0.26/install-aws-standard.md:126]: Since these need to be in order, should we keep using a numbered list here?

**Review by jwitz — APPROVED** [review]
One small comment, otherwise LGTM! I noticed that these edits only seem to be for the 0.26 version of Software documentation, can you also make these changes in the base `software` directory? Don't worry about other versions, as customers will only ever be installing the most recent version of Software.

**Comment by kwcanuck** [comment]
@jwitz I have added the edits to the base `software` directory as requested. Thanks for the reminder!

A numbered list should only be used for a task list. In this example, the user is asked to "verify the order of the certificates", so it is clear that the items should appear in the order specified. In my opinion, there is no need to use a numbered list.

**Comment by jwitz** [comment]
Makes sense to me! Good call, and thanks for explaining it to me. Agreed that we can keep it as bullets

**Pull request merged**